src/clients/starknet-tx/index.ts

```diff
@@ -1,34 +1,55 @@
-import { AddTransactionResponse, Contract, defaultProvider as provider, hash } from 'starknet';
+import {
+  Account,
+  AddTransactionResponse,
+  Contract,
+  defaultProvider as provider,
+  hash
+} from 'starknet';
 import abi from './abi/auth.json';
 import constants from './constants.json';
 import { strToShortStringArr } from '../../utils/strings';
 
 const { getSelectorFromName } = hash;
 
 export class StarkNetTx {
+  public auth: Contract;
+  public account: Account;
+  constructor(_account: Account) {
```

Comment:
There should be a new line here
```suggestion
  public account: Account;
  
  constructor(_account: Account) {
```